ve026a:
  controller:
    state:
      type: joint_state_controller/JointStateController
      publish_rate: 50
    position:
      gripper_controller:  
        type: position_controller/JointPositionController
        joints:
          - joint_7       
      arm_joint_controller:
        type: position_controller/JointPositionController
        joints:
          - joint_1
          - joint_2
          - joint_3
          - joint_4
          - joint_5
          - joint_6
    trajectory:
      arm_position_controller:
        type: position_controllers/JointTrajectoryController
        joints:
          - joint_1
          - joint_2
          - joint_3
          - joint_4
          - joint_5
          - joint_6
        gains:
          joint_1: {p: 1.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_2: {p: 1.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_3: {p: 1.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_4: {p: 1.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_5: {p: 1.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_6: {p: 1.0, i: 1.0, d: 0.1, i_clamp: 1.0}       
      arm_velocity_controller:
        type: velocity_controllers/JointTrajectoryController
        joints:
          - joint_1
          - joint_2
          - joint_3
          - joint_4
          - joint_5
          - joint_6
        gains:
          joint_1: {p: 100.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_2: {p: 100.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_3: {p: 100.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_4: {p: 100.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_5: {p: 100.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_6: {p: 100.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
      arm_velocity_feedforward_controller:
        type: velocity_controllers/JointTrajectoryController
        joints:
          - joint_1
          - joint_2
          - joint_3
          - joint_4
          - joint_5
          - joint_6
        gains:
          joint_1: {p: 10.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_2: {p: 10.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_3: {p: 10.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_4: {p: 10.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_5: {p: 10.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
          joint_6: {p: 10.0, i: 1.0, d: 0.1, i_clamp: 1.0} 
        velocity_ff:
          joint_1: 1.0
          joint_2: 1.0
          joint_3: 1.0
          joint_4: 1.0
          joint_5: 1.0
          joint_6: 1.0
